task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +529,31 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+    let winCoords = [
+        [11,12,13], [21,22,23], [31,32,33],
+        [11,21,31], [12,22,32], [13,23,33],
+        [11,22,33], [13,22,31]
+    ];
+    // Привет Алекс, не универсально но грубо говоря я скоротил решение на половину узнав выигрышные координаты заранее.
+    // Какая логика?
+    // Проверяем заранее известные координаты победы.
+    // 11 12 13
+    // Логика проверки
+    // Каждый раз нам придётся сверять по 3 отметки, если хотя бы 1 из комбинации является выигрышной значит это победа одного из маркеров
+    let n = 3;
+    let firstLetter="";
+    for(let i=0;i<winCoords.length;i++){
+        let mark=0;
+        firstLetter=position[Number(winCoords[i][0].toString()[0])-1][Number(winCoords[i][0].toString()[1])-1];
```

Comment:
this is not very readable why don't just code win positions as
[[1,1],[1,2],[1,3]]
?

or just write a function that gets element form position array

Reply:
because this is i))

Reply:
yeap, but keep in mind that somebody's going to read it